src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/AboutToSubmitCallbackResponse.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.ccd.domain.service.stdapi;
+
+import uk.gov.hmcts.ccd.domain.model.callbacks.SignificantItem;
+
+import java.util.Optional;
+
+public class AboutToSubmitCallbackResponse {
+
+    private Optional<String> callBackResponse = Optional.empty();
```

Comment:
Name of this property is wrong, this should be called `state` as it's an optional state returned to override the event's post state